modules/SignalCalculator.py

```diff
@@ -349,9 +357,9 @@ def _calcInductionVoltages(self):
                         part_magnetic_charge= self._part_magnetic_charge,
                         part_magnetic_moment= self._part_magnetic_moment
             )
-            self._sample_times[index]           += sample_times
-            self._voltages_wo_noise[index]      += voltages
-            self._voltages[index]               += voltages
+            self._sample_times[index]           = sample_times
```

Comment:
Is this where the bug was? Or is it a minor one?

Reply:
this is the minor one, i change this to avoid sample_times list being totally zero, which will lead None result in fft calculation